lib/services/local/presence_test.go

```diff
@@ -191,3 +191,65 @@ func TestDatabaseServersCRUD(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, 0, len(out))
 }
+
+func TestNodeCRUD(t *testing.T) {
```

Comment:
These tests are no longer relevant since I removed the backend implementation of `ListNodes`, but I figured I'd just leave the others here.